packages/rancher-monitoring/generated-changes/patch/templates/grafana/dashboards-1.14/controller-manager.yaml.patch

```diff
@@ -15,16 +15,20 @@
    name: {{ printf "%s-%s" (include "kube-prometheus-stack.fullname" $) "controller-manager" | trunc 63 | trimSuffix "-" }}
    annotations:
  {{ toYaml .Values.grafana.sidecar.dashboards.annotations | indent 4 }}
-@@ -107,7 +108,7 @@
+@@ -107,7 +108,11 @@
                          "tableColumn": "",
                          "targets": [
                              {
 -                                "expr": "sum(up{job=\"kube-controller-manager\"})",
++                                {{- if .Values.k3sServer.enabled  }}
++                                "expr": "sum(up{job=\"{{ include "exporter.kubeControllerManager.jobName" . }}\", metrics_path=\"/metrics\"})",
```

Comment:
@dbason I copied this from your commit, but why do we need it?

Reply:
Because with k3s we include all of the metrics in the same job.  Now that we're also scraping the kubelet metrics we have 3 sets of paths; `/metrics` `/metrics/cadvisor` and `/metrics/probes`.  If we don't do that specify the metrics_path it triples the count in the metric box in the dashboard